libtock2/examples/console.rs

```diff
@@ -0,0 +1,15 @@
+//! An extremely simple libtock-rs example. Just prints out a message
+//! using the Console capsule, then terminates.
+
+#![no_main]
+#![no_std]
+use core::fmt::Write;
+use libtock2::console::Console;
+use libtock2::runtime::{set_main, stack_size};
+
+set_main! {main}
+stack_size! {0x100}
+
+fn main() {
+    writeln!(Console::writer(), "Hello world!").unwrap();
```

Comment:
In typical apps, I assume we would only want to call `Console::writer()` once, and then pass that writer to everywhere it was needed? If so, perhaps we should store it in a `writer` variable here and pass that to `writeln!()`, so that people copying from this example are nudged toward the better approach in the case of multiple writes?

Reply:
I see. I suppose the call is very cheap, I am fine leaving it as-is